README.md

```diff
@@ -968,6 +969,82 @@ The global default Topic authorizations are expressed in the project schema and
 parameter "includeAuthorization" equal to "true" Topics will include an "authorization" field containing any local
 overrides for each Topic.
 
+### 4.2.9 GET Topic History Service
+
+**Resource URL**
+
+    GET /bcf/{version}/projects/{project_id}/topics/history
+
+[topic_history_GET.json](Schemas_draft-03/Collaboration/History/topic_history_GET.json)
+
+Retrieve a **collection** of topic histories related to a project (default sort order is `date`).
+
+**Topic history types**
+
+|type|value|
+|---------|-----------|
+|topic_created|null|
+|title_set|The title (limit: 100 characters)|
+|description_set|The description (limit: 500 characters)|
+|status_set|The status (value from extensions) |
+|type_set|The type (value from extensions)|
+|priority_set|The priority (value from extensions)|
+|assigned_set|The assigned user (value from extensions)|
+|label_added|The added label (value from extensions)|
```

Comment:
When multiple (e.g. 3) labels are added/removed at the same time, does it result in multiple (e.g. 3) history records with each one specifying a single label added/removed then? Or alternatively, should the "value" field in the response JSON should be an array instead of a single string (i.e. in "topic_history_GET.json", instead of "value": {"type": ["string","null"]}, should it be "value": {"type": ["array","null"]})?